modules/apps/object/object-test/src/testFunctional/tests/ObjectNotifications.testcase

```diff
@@ -563,7 +563,6 @@ definition {
 	@description = "LPS-178783 - Verify a standalone action can be triggered manually to send an email"
 	@priority = 5
 	test CanSendEmailViaActionStandalone {
-		property custom.properties = "feature.flag.LPS-173894=true";
```

Comment:
This feature flag was removed. Please [see](https://github.com/brianchandotcom/liferay-portal/pull/132700/commits/ddb19911bf43e531c3861c6c5254ae441bf63e22#diff-e652d04e2647ab1ad8d34df89282488f6088a1b238474173afcca9b01484eb63L6576-L6580)